tests/connect/test_basic_column.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from pyspark.sql.functions import col
+from pyspark.sql.types import StringType
+
+
+def test_column_operations(spark_session):
+    # Create DataFrame from range(10)
+    df = spark_session.range(10)
+
+    # Test __getattr__
+    # todo: https://github.com/Eventual-Inc/Daft/issues/3433
+    # df_attr = df.select(col("id").desc())  # Fix: call desc() as method
+    # assert df_attr.toPandas()["id"].iloc[0] == 9, "desc should sort in descending order"
+
+    # Test __getitem__
+    # todo: add extract value
+    # df_item = df.select(col("id")[0])
+    # assert df_item.toPandas()["id"].iloc[0] == 0, "getitem should return first element"
+
```

Comment:
Nit: let's move each individual test to its own unit test? Makes it easier to debug if something goes wrong so we can see them as separate unit test failures.